Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
     "components/zcash_encoding",
     "components/zcash_protocol",
     "components/zip321",
-#    "zcash_client_backend",
+    "zcash_client_backend",
 #    "zcash_client_sqlite",
```

Comment:
Let's add a comment above line 9
`# zcash_client_backend and zcash_client_sqlite are not compatible with OrchardZSA`